deepfence_server/handler/integration.go

```diff
@@ -247,7 +254,14 @@ func (h *Handler) UpdateIntegration(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if !obj.IsValidCredential(ctx) {
+	vc, err := obj.IsValidCredential(ctx)
+	if err != nil {
+		log.Error().Msgf("%v", err)
+		h.respondError(&ValidatorError{err: err}, w)
```

Comment:
ValidatorError requires a certain format to show error in the specific UI form element.
```
errUserInactive       = ValidatorError{
		err: errors.New("email:user is not active")
	}
```
We can make it `BadDecoding`, which will show error below the form.

Reply:
makes sense.

Reply:
done